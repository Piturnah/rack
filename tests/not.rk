// `not` acts as boolean inverter. Should get `0 1`.

true not print
false not print