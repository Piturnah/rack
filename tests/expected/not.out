----STDOUT----
0
1

----STDERR----
